src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -66,14 +66,15 @@ internal TokenCache TokenCache
         protected RequestBase(
             IServiceBundle serviceBundle,
             AuthenticationRequestParameters authenticationRequestParameters,
-            ApiEvent.ApiIds apiId)
+            ApiEvent.ApiIds apiId,
+            bool validateScopes)
         {
             ServiceBundle = serviceBundle;
             TokenCache = authenticationRequestParameters.TokenCache;
             _apiId = apiId;
             
             AuthenticationRequestParameters = authenticationRequestParameters;
-            if (authenticationRequestParameters.Scope == null || authenticationRequestParameters.Scope.Count == 0)
+            if (validateScopes && (authenticationRequestParameters.Scope == null || authenticationRequestParameters.Scope.Count == 0))
             {
                 throw new ArgumentNullException(nameof(authenticationRequestParameters.Scope));
             }
```

Comment:
should we ever throw an exception in this case? I think we should support it, afaik, this is not a spec requirement. In some scenarios you may not get an AT back if it's not specified. But you will get the idt which can be used to build AuthZ

Reply:
the validateScopes will be removed